### ISSUE TITLE
Contestant communications on RWS

### ISSUE THREAD

**Issue by rspencer01** [cms-dev/cms#456]
Is it possible/in scope to have the RWS show the global communications sent to contestants?  This way, if, for example, a global communication saying "Problem 2 has been changed and re-evaluated" would be available to spectators to explain score changes.

**Comment by stefano-maggiolo** [comment]
I don't think that an unfiltered view of the announcements is something we will ever implement, but a way of displaying "news" specific for RWS is useful I think, exactly for the reason you say.

Other examples:
- during the IOI we (I) made a few mistakes in handling the scoreboard and there were a few times in which it wasn't correct, and it would be nice to say it afterwards;
- the evaluation may terminate later than the contest, and there is no way for users to notice when it is finished;
- some contestants had additional time, which confused people seeing submissions after the 5h mark.

**Comment by prandla** [comment]
This is basically the same thing as #649.
